api/v3/Job.php

```diff
@@ -378,6 +378,19 @@ function _civicrm_api3_job_fetch_bounces_spec(&$params) {
 }
 
 /**
+ * Metadata for activities function.
+ *
+ * @param array $params
+ */
+function _civicrm_api3_job_fetch_activities_spec(&$params) {
+  $params['is_create_activities'] = array(
+    'api.default' => 0,
```

Comment:
No, just thinking - I think it should not be a param but rather be always set in the _fetch_activities function - since there is no point calling that fn without creating activities